acceptance/analyzer_test.go

```diff
@@ -47,8 +47,8 @@ func TestAnalyzer(t *testing.T) {
 	analyzeDaemonFixtures = analyzeTest.targetDaemon.fixtures
 	analyzeRegFixtures = analyzeTest.targetRegistry.fixtures
 
-	for _, platformAPI := range api.Platform.Supported {
-		spec.Run(t, "acceptance-analyzer/"+platformAPI.String(), testAnalyzerFunc(platformAPI.String()), spec.Parallel(), spec.Report(report.Terminal{}))
+	for _, platformAPI := range []string{"0.12"} {
```

Comment:
```suggestion
	for _, platformAPI := range api.Platform.Supported {
```
Aye, I'm afraid we do need to run the analyzer tests for all supported platforms. I think we skip over older platforms in the extender acceptance tests, but that is only because the extender isn't supported on older platforms.